### PR TITLE
fix: tests in `helios-ethereum`

### LINKED ISSUE #415: fix: tests failing in `helios-ethereum`
Unit tests for `ConsensusClient` expect more generic arguments

### PR DESCRIPTION
**Pull request by 0xRampey**
fixes #415

### CODE REVIEW

File: ethereum/consensus-core/src/consensus_core.rs [hunk 1]
```diff
@@ -307,6 +307,8 @@ pub fn verify_generic_update<S: ConsensusSpec>(
         } else {
             return Err(ConsensusError::InvalidFinalityProof.into());
         }
+    } else {
+        return Err(ConsensusError::MissingFinalizedHeader.into());
```
Comment (ncitron) [inline ethereum/consensus-core/src/consensus_core.rs:311]: It is possible to have an update that does not contain a finalized header (see optimisitc updates).

Comment (0xRampey) [reply ethereum/consensus-core/src/consensus_core.rs:311]: Ah, right

**Comment by ncitron** [comment]
Can you run `cargo fmt` so that the fmt CI passes?

**Comment by 0xRampey** [comment]
Done!

**Review by ncitron — APPROVED** [review]
LGTM!

**Pull request merged**